RealmSwift/Sync.swift

```diff
@@ -231,6 +231,10 @@ public struct SyncConfiguration {
     }
 }
 
+#if canImport(Combine)
+import Combine
+#endif
```

Comment:
Thís import seems unnecessary.

Reply:
we need it to handle Functions' dynamicLookup with Future